### ISSUE TITLE
[Feature request] Displayed latest change in a file/latest release

### ISSUE THREAD

**Issue by Chirunos** [ChillerDragon/ChromeGithub#1]
It would be very useful to find only updated forks of (mainly discontinued) repositories

**Comment by ChillerDragon** [comment]
I am not sure if I understand what you want. Do you mean appending the latest commit message of the default branch after every fork in the list? Or should it sort by date of last activity?

**Comment by Chirunos** [comment]
Yes, the latest commit message of the default branch

**Comment by ChillerDragon** [comment]
Turned out to be more work than I expected or I am just being stupid again. Seems like those commit messages go through some javascript shenanigans. I managed to get a half working version but only a few commit messages actually go through. Will look into it maybe someday if I am bored but rage quit for now hehe.

If you are just interested in forks that added more commits just have a look for the green number those are commits ahead of upstream.

![Screenshot from 2022-10-10 14-47-54](https://user-images.githubusercontent.com/20344300/194870067-9cde3b39-bba6-4bf4-8558-84d2dc8ad562.png)
